src/System.Management.Automation/engine/hostifaces/MshHost.cs

```diff
@@ -296,6 +296,11 @@ public interface IHostSupportsInteractiveSession
         /// <summary>
         /// Called by the engine to notify the host that a runspace push has been requested.
         /// </summary>
+        /// <param name="runspace">
+        /// The runspace to push. This runspace must be a remote runspace and
+        /// not a locally created runspace.
```

Comment:
Since it is API comment I'd explicitly point it should be `RemoteRunspace`.

Reply:
`RemoteRunspace` is not a public type so I didn't think of actually using the proper type name.

Reply:
I think it would be clearer, but I don't insist.

Reply:
I think I'll leave it off unless others also prefer it in there. I'm unsure how the docs would actually be rendered if a non-public type was referenced here.